src/plugins/intel_gna/frontend/quantized_layer_params.hpp

```diff
@@ -78,14 +78,15 @@ class Quantization {
 };
 
 struct QuantizedLayerParams {
-    Quantization _src_quant;
-    Quantization _dst_quant;
+    QuantizationParams _src_quant;
```

Comment:
We are using google code style, and class members should be named like src_quant_  (underscore at end) but it maybe one more exception in OV, @nkogteva do you know?

Reply:
You're right, I put leading instead of trailing underscore. But actually there is no agreement on what we should actually use in several cases. :-)

Reply:
@mryzhov I propose run ncc tool and fix code style in separate PR

Reply:
ok